pinax/api/jsonapi.py

```diff
@@ -77,16 +77,16 @@ def get_serializable_data(self, request=None):
                     included=self.included,
                     request=request,
                 ))
-            return ret
+            return ret, dict(total_count=len(self.data))
```

Comment:
No need to return the `meta` dictionary to get updated in `self.meta`. Just update here. The object state is the top-level. There are some other improvements that should be made to support that, but out of scope for this PR.